tests/testproject/TestProject.IntegrationServiceA/SqlServer/SqlServerExtensions.cs

```diff
@@ -13,14 +16,16 @@ public static void MapSqlServerApi(this WebApplication app)
 
     private static async Task<IResult> VerifySqlServerAsync(SqlConnection connection)
     {
+        StringBuilder errorMessageBuilder = new();
```

Comment:
In a follow up PR I will add listening to logs from the services in `AppHost`, so this shouldn't be needed then.